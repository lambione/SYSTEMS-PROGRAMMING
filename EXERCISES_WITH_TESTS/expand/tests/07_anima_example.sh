#!/bin/bash
exec "$1" tests/anima_rules.txt <<EOF
ANIMA
SANITY
ANIMA SANITY
SANITY ANIMA
EOF
