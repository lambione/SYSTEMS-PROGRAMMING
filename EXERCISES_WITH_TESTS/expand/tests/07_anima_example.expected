ASANISIMASA
TY
ASANISIMASA TY
TY ASANISIMASA
